docs/src/configs/docs/records/entities.js

```diff
@@ -42,6 +42,18 @@ export default [
       'All the possible values for the "period" field of the record model. (Note: a month is 31 days)',
     values: ['6h', 'day', 'week', 'month', 'year']
   },
+  {
+    name: 'Player types',
+    description:
+      'All the possible values for the "playerType" field of the record model.',
```

Comment:
There is no playerType in the record model, maybe change this to `...for the "playerType" query parameter.`

**The same applies for the playerBuild below.**

Reply:
Fixed in ae1a27c